### ISSUE TITLE
Transforming `back()` function using HelperFunctionToConstructorInjectionRector in Laravel works incorrectly

### ISSUE THREAD

**Issue by sashabeton** [rectorphp/rector#1824]
PHP version: 7.2
Full Command: `rector process ./src`

## Current Behaviour 

`back()` function is being transformed incorrectly (without function usage)

Source:
```php
<?php

use Illuminate\Http\RedirectResponse;

class Controller
{
    public function someAction(): RedirectResponse
    {
        return back();
    }
}

```

Transformed:

```php
<?php

use Illuminate\Http\RedirectResponse;
use Illuminate\Routing\Redirector;

class Controller
{
    /** @var Redirector */
    private $redirector;

    public function __construct(Redirector $redirector)
    {
        $this->redirector = $redirector;
    }

    public function someAction(): RedirectResponse
    {
        return $this->redirector;
    }
}

```

## Expected Behaviour

Expected behavior is to transform `back()` call correctly (`$this->redirector->back()`)

![image](https://user-images.githubusercontent.com/23243361/62630958-945de780-b938-11e9-9fd5-79e2824603c5.png)

**Comment by TomasVotruba** [comment]
Thanks for reporting the issue.

Please send PR with failing test case, so we can spot the exact point of the error

**Comment by TomasVotruba** [comment]
Resolved by #1845

@sashabeton Just a tip to let Github close this automatically: https://help.github.com/en/articles/closing-issues-using-keywords
